src/sidebar/util/time.ts

```diff
@@ -48,56 +47,46 @@ function format(date, options, Intl = window.Intl) {
 }
 
 /**
- * @callback DateFormatter
- * @param {Date} date
- * @param {Date} now
- * @param {Intl} [intl]
- * @return {string} formatted date
+ * @return formatted date
  */
+type DateFormatter = (date: Date, now: Date, intl?: IntlType) => string;
 
-/** @type {DateFormatter} */
-function nSec(date, now) {
+const nSec: DateFormatter = (date, now) => {
   const n = Math.floor(delta(date, now) / SECOND);
   return `${n} secs ago`;
-}
+};
```

Comment:
Changing these named functions to typed variables with a common type seems like a reasonable choice in this context.

Reply:
Oh, yeah. I forgot to add a comment explain why I changed it.

Since they were not exported from this module, the impact was low.